api/src/main/java/com/codeforcommunity/exceptions/ExistingTeamRequestException.java

```diff
@@ -0,0 +1,29 @@
+package com.codeforcommunity.exceptions;
+
+import com.codeforcommunity.rest.FailureHandler;
+import io.vertx.ext.web.RoutingContext;
+
+public class ExistingTeamRequestException extends RuntimeException implements HandledException {
```

Comment:
Could be more specific. Is this for when a user requests to join a team, but they have already done so?